README.md

```diff
@@ -1,5 +1,7 @@
 # PublishTestResultScreenshot for Azure DevOps
-Get more insight on your UI test failure with screenshots. 
+**DEPRECATED**: This Azure DevOps extension is no longer maintained.
```

Comment:
Please add a link to the alternative: https://devblogs.microsoft.com/devops/junit-attachments-support-for-publish-test-results/

Reply:
Mention of alternative way for upload file attachments included